### DISCUSSION TITLE
3D prints

### DISCUSSION THREAD

**Discussion by CooperTrout** [frc5567/2023-Robot#121] [Team Posts]
I have finished 3D printing the Pixy camera case.
What kind of Arduino do we have so I can try and find a case for that and do you want me to try and find a navx Mxp case to print?

**Comment by MikasaSucasa** [comment]
We have an arduino mega. I think that the pixy and mega are more important to finish but we might like the mavx to be covered eventually.
Thanks Cooper!

**Comment by coverbeek** [comment]
The Navx MXP might be nice, since Thomas was suggesting moving the Rio to make it more accessible, and needing to use the ribbon cable to connect the navx so it was still relatively centered. It would be good to have it in somewhat of a protective case (but it needs to support the ribbon cable if we do so.

-Chris
